roboconf-dist-rpm-agent/src/rpm/control/postinst

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+#
+#  Copyright 2014 Linagora, Université Joseph Fourier
```

Comment:
No. We will most likely have to change (c) headers everywhere next week.
So, please, keep the mention to Université Joseph Fourier. Université Grenoble-alpes inherits, by definition, of the IP.